extensions/amp-story/amp-story.md

```diff
@@ -1386,6 +1386,177 @@ The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON confi
   ]
 }
 
+```
+## Experimental Feature: Branching
+
+We’ve implemented additional features to allow for the construction of non-linear stories. By utilizing these features, you can create stories that start from somewhere other than the beginning, share specific pages of stories, or even allow users to jump around within a story.  Example use cases include quizzes, choose-your-own-story experiences, and story table of contents.
+
+Note that all branching features are behind an `amp-story-branching` experiment. To turn on the experiment, enable it in your dev console:
```

Comment:
Strike "Note that" and start with "All branching..."

Reply:
Done